### PR TITLE
removes old template code RenderLandingPage.js and unused route '/landing'

### PR DESCRIPTION
**Pull request by jglez**
# Code Cleanup

## What was changed
remove
remove2
remove3

- Removed `src/components/pages/Landing` directory
  - Removed LandingContainer.js
  - Removed RenderLandingPage.js
 
- Removed import of RenderLandingPage.js in `index.js`
- Removed old route "/landing" for RenderLandingPage.js

## Type of change
- Removal of dead code/code cleanup
- Removed unused paths and components
- Removed vestigial template code

### CODE REVIEW

**Review by fullstackcaveman — APPROVED** [review]
Great job removing unneeded items form the codebase!

File: src/index.js [hunk 1]
```diff
@@ -17,7 +17,6 @@ import { ExampleListPage } from './components/pages/ExampleList';
 import { ProfileListPage } from './components/pages/ProfileList';
 import { LoginPage } from './components/pages/Login';
 import { HomePage } from './components/pages/Home';
-import { LandingPage } from './components/pages/Landing';
```
Comment (nicholasbus) [inline src/index.js:20]: Always good to remove unused code 😄

File: src/index.js [hunk 2]
```diff
@@ -60,7 +59,6 @@ function App() {
         <Route path="/login" component={LoginPage} />
         <Route path="/signup" component={SignupPage} />
         <Route path="/implicit/callback" component={LoginCallback} />
-        <Route path="/landing" component={LandingPage} />
```
Comment (nicholasbus) [inline src/index.js:63]: Removing unneeded code little by little 🙏

**Review by nicholasbus — APPROVED** [review]
Very nice. removing unused code is definitely something that is needed in this code base.

**Pull request merged**